docker-compose.yml

```diff
@@ -57,9 +57,63 @@ services:
     networks:
       - celestia-zkevm-net
 
+  rollkit-evm-single:
+    image: ghcr.io/rollkit/rollkit-evm-single:v0.1.3
+    container_name: rollkit-evm-single
+    entrypoint: /usr/bin/entrypoint.sh
+    environment:
+      - EVM_ENGINE_URL=http://reth:8551
+      - EVM_ETH_URL=http://reth:8545
+      - EVM_JWT_SECRET=f747494bb0fb338a0d71f5f9fe5b5034c17cc988c229b59fd71e005ee692e9bf
+      - EVM_GENESIS_HASH=0x06bab65f0fa3c9b1ba0da34e319d7a5cd948cd10ba33080b88f17d281880e344
+      - EVM_BLOCK_TIME=1s
+      - EVM_SIGNER_PASSPHRASE=secret
+      - DA_ADDRESS=http://celestia-bridge:26658
+      - DA_AUTH_TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJBbGxvdyI6WyJwdWJsaWMiLCJyZWFkIiwid3JpdGUiLCJhZG1pbiJdfQ.qxoN_a-k7aZZE77VSpfgyvI_0eNvJ_OgBsT35zxUQnQ
+      - DA_NAMESPACE=00000000000000000000000000000000000000b7b24d9321578eb83626
+    depends_on:
+      celestia-bridge:
+        condition: service_healthy
+      reth:
+        condition: service_started
+    networks:
+      - celestia-zkevm-net
+  
+  reth:
+    image: ghcr.io/paradigmxyz/reth:v1.3.11
+    container_name: reth
+    ports:
+      - "9001:9001"   # Metrics
+      - "30303:30303" # P2P
+      - "8545:8545"   # HTTP
+      - "8551:8551"   # Auth RPC (Engine API)
+    volumes:
+      - ./testnet/reth:/home/reth
+      - reth:/home/reth/eth-home
```

Comment:
Minor tweaks here:
- Bind mount for genesis file and jwt for engine api
- Named docker volume for reth state dir